Modules/xASL_module_ASL.m

```diff
@@ -210,6 +206,32 @@
     warning('x.Q didnt exist, are quantification parameters lacking?');
 end
 
+%% Multi-PLD parsing
+
+
+%% TimeEncoded parsing
+
+% Check if TimeEncoded is defined
+if isfield(x,'TimeEncodedMatrixType') || ~isempty(x.TimeEncodedMatrixType)
```

Comment:
Should this be `||` or `&&` / :) Same below!

Reply:
Probably `&&`, right?

Reply:
yes of course sorry, done